pallets/committee/src/lib.rs

```diff
@@ -433,7 +466,7 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
     /// Given `votes` number of votes out of `total` votes, this function compares`votes`/`total`
     /// in relation to the threshold proporion `n`/`d`.
     fn is_threshold_satisfied(votes: u32, total: u32, (n, d): (u32, u32)) -> bool {
-        votes * d >= n * total
+        votes * d > n * total
```

Comment:
Hmm; what happened here? If we have `votes: 1, total: 2, (n, d): (1, 2)` then before we got:
```rust
1 * 2 >= 1 * 2 ==> true
```
and now:
```rust
1 * 2 > 1 * 2 ==> false
```

Reply:
See the changed behaviour in the description. Passing a proposal with 1/2 members voting for and 1/2 against is incorrect.

Reply:
I don't know; it seems like a design decision and not clearly incorrect. Might be better to do such changes in a separate PR though rather than in a benchmark PR.

Also, the `type CommitteeOrigin`s in the `runtime.rs` specs use `EnsureProportionAtLeast` rather than `EnsureProportionMoreThan`. Maybe that doesn't matter, but it seems like things are inconsistent now and the former type should be removed. The docs of `set_vote_threshold` should also be reviewed as a result.

Reply:
The benchmark depends on the way we compute when to pass a proposal. So maybe a better way would be to use `EnsureProportion`s in `is_threshold_satisfied`.

Whether passing in a draw is correct or not is another issue, I agree. IMHO since the benchmarks happen to depend on a committee instance with the `== 1/2 members` proportion, it would be great to solve it now rather than postpone and then have to update the benchmarks.

Reply:
Reverted.